<testsuite name="OSMO Test Suite" timestamp="01.01.1970 00:00:01" hostname="OSMO rules" tests="1" failures="0" errors="1" time="2.00">
  <properties>
    <property name="Algorithm" value="osmo.tester.generator.algorithm.RandomAlgorithm"/>
    <property name="Listener" value="osmo.tester.reporting.jenkins.JenkinsReportGenerator"/>
    <property name="Model Object" value="osmo.tester.unittests.testmodels.ErrorModel5"/>
    <property name="Seed" value="333"/>
    <property name="Suite End Condition" value="And{conditions=[Length{length=1}, Probability{threshold=0.05}]}"/>
    <property name="Test Case End Condition" value="And{conditions=[Length{length=1}, Probability{threshold=0.1}]}"/>
  </properties>
  <testcase name="Test1" classname="[osmo.tester.unittests.testmodels.ErrorModel5]" time="0.00">
    <error message="string" type="string">java.lang.Exception: java.lang.AssertionError: @TestStep assert fail
	at osmo.tester.reporting.jenkins.JenkinsReportGenerator.testError(JenkinsReportGenerator.java:132)
	at osmo.tester.generator.listener.GenerationListenerList.testError(GenerationListenerList.java:108)
	at osmo.tester.generator.MainGenerator.handleError(MainGenerator.java:221)
	at osmo.tester.generator.MainGenerator.nextTest(MainGenerator.java:161)
	at osmo.tester.generator.MainGenerator.generate(MainGenerator.java:94)
	at osmo.tester.OSMOTester.generate(OSMOTester.java:90)
	at osmo.tester.unittests.reporting.jenkins.ReportTests.testError(ReportTests.java:182)
	at osmo.tester.unittests.reporting.jenkins.ReportTests.error5Tests(ReportTests.java:165)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	at com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:47)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:242)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)
Caused by: java.lang.AssertionError: @TestStep assert fail
	at osmo.tester.unittests.testmodels.ErrorModel5.transition1(ErrorModel5.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at osmo.tester.model.InvocationTarget.invoke(InvocationTarget.java:39)
	at osmo.tester.generator.MainGenerator.execute(MainGenerator.java:291)
	at osmo.tester.generator.MainGenerator.nextStep(MainGenerator.java:272)
	at osmo.tester.generator.MainGenerator.nextTest(MainGenerator.java:155)
	... 27 more
</error>
  </testcase>
  <system-out></system-out>
  <system-err></system-err>
</testsuite>

